src/lib/chat/matrix-client.ts

```diff
@@ -231,13 +231,22 @@ export class MatrixClient implements IChatClient {
       const relatedEventId = this.getRelatedEventId(editEvent);
       const messageIndex = messages.findIndex((msg) => msg.id === relatedEventId);
       if (messageIndex > -1) {
-        const newContent = this.getNewContent(editEvent);
-        if (newContent) {
+        if (editEvent.content.msgtype === MatrixConstants.BAD_ENCRYPTED_MSGTYPE) {
```

Comment:
I feel like this whole function is getting out of hand. Let's refactor this before adding new functionalty.

Reply:
https://github.com/zer0-os/zOS/pull/1210. I will update this branch with those changes.